### PR TITLE
[#178958851] Upload the Dandiset

### PR DESCRIPTION
**Pull request by dsleiter**
https://www.pivotaltracker.com/story/show/178958851

This update documents the steps used to upload the dandiset, as well as the metadata fields which were added.

Note: this should be reviewed only after acceptance of #3 in order to minimize the diff with master.

### CODE REVIEW

**Comment by dsleiter** [comment]
@sumner15 please wait to review this until #2 and #3 are merged. I'll let you know when to take a look at it.

**Comment by dsleiter** [comment]
@sumner15 ok, this one is ready for review! Not too much to review - most just documentation, but I wanted to get your eyes on it.

File: README.md [hunk 1]
```diff
@@ -78,3 +78,61 @@ Some additional data is mapped from BIDS metadata files:
     * `"TaskName"` -> `NWBFile.notes`
     * `"PowerLineFrequency"` -> `NWBFile.notes`
 * `subj-{i}/nirs/subj-{j}_task-tapping_events.tsv` -> `TimeSeries` in `NWBFile.stimulus`
+
+
+## Uploading dataset to the DANDI archive
```
Comment (sumner15) [inline README.md:83]: This section is great. I ran through most of it (not uploading of course) 👍🏼

**Review by sumner15 — APPROVED** [review]
Looks great!

**Comment by sumner15** [comment]
And congrats again getting this pushed up to DANDI! 🎉

**Pull request merged**